src/controllers/travelControllers.js

```diff
@@ -21,4 +21,26 @@ export default {
             errorResponse(res, statusCode.serverError, err);
         }
     },
+
+    pendingManagerApproval: async(req, res) => {
+        const { role } = req.userData;
+
+        const { manager } = req.params;
+
+        if (role == 'user') {
```

Comment:
I don't think the double = for equality is ideal. Why not use the strict equality operator (===)?